test/index.test.js

```diff
@@ -11,17 +13,173 @@ QUnit.test('the environment is sane', function(assert) {
                      'brightcovePlayerLoader is a function');
 });
 
-QUnit.module('brightcove-player-loader', {
+QUnit.module('brightcove-player-loader', function(hooks) {
+  const originalBaseUrl = getBaseUrl();
 
-  beforeEach() {
-    this.clock = sinon.useFakeTimers();
+  hooks.before(function() {
+    setBaseUrl('http://localhost:9999/test/fixtures/');
```

Comment:
this should probably use `window.location.origin` since the port can change.

Reply:
Good point, heh.